applications/nrf_desktop/configuration/common/led_state.h

```diff
@@ -75,7 +75,7 @@ enum led_id_nrf_desktop {
 #endif
 
 #if defined(CONFIG_BT_PERIPHERAL)
-#define LED_PEER_COUNT (CONFIG_BT_MAX_PAIRED - 1)
+#define LED_PEER_COUNT (CONFIG_BT_ID_MAX - 2)
```

Comment:
why '2'?

Reply:
We cannot use the default local identity (we cannot reset it) and one local identity is used only for erase advertising.

Reply:
I will add definitions to make it more clear.